rom-hacking/scripting/animation-commands/ac-mark-gyoe.md

```diff
@@ -0,0 +1,27 @@
+# (783) _AC_MARK_GYOE
+
+## Effect
+
+Displays an exclamation mark bubble \[!\] above the actors head.
+
+Sets _moveEndTime to 1.5.
+
+## Syntax
+
+```c
+_AC_MARK_GYOE(unknown?)
+```
+
+| Argument | Description | Types | Required |
+| - | - | - | - |
+| **unknown** | Always 1 when present | Float | Optional |
```

Comment:
I have not seen any use of this argument in exefs, so I think it's safe to just remove it.